scripts/loadable_app/loadable_app.sh

```diff
@@ -0,0 +1 @@
+gcc hello2.c -o hello -nostdlib -emain -fpie -N -m32 "$@"
```

Comment:
There is no `hello2.c` in PR, only `hello.c`

Since there is `"$@"` argument already, I suggest to remove explicit input and output file
```
gcc -nostdlib -emain -fpie -N -m32 "$@"
```
Then this script could be used with arbitrary sources and executables, from any current working dir. For example, it could be used from topdir like
```
./scripts/loadable_app/loadable_app.sh ./scripts/loadable_app/hello.c -o conf/rootfs/hello
```
I'd be great to have comment on top of this script with some info and suggested usage (like mine example above)